meson.build

```diff
@@ -375,12 +375,12 @@ if gettext
   config_data.set('ENABLE_GETTEXT', 1)
 endif
 
-configure_file(
+config_h = configure_file(
   output: 'config.h',
   configuration: config_data,
 )
 
-compiler_common_flags += ['-include', 'config.h']
+compiler_common_flags += ['-include', config_h.full_path()]
```

Comment:
You don't need to bump the minimum required version. You do "need" the full path if polkit can ever be built as a subproject -- the compiler is always run from the global_build_root(), that is, the same directory as the `build.ninja` file, and config.h is generated where current_build_dir() == project_build_root(), which is the same directory as `build.ninja` except in a subproject.

But regardless, the string `config_h.full_path()` is provided for API equivalence with other objects. You could have always reliably guaranteed the location quite simply:

```meson
config_h_full_path = meson.current_build_dir() / 'config.h'
```

No version bump needed.

Reply:
Ah, that's actually pretty neat, thanks a lot for the explanation!